components/tabs/tabs.js

```diff
@@ -291,10 +292,8 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 				}
 
 				const expanded = await this._tryExpandTabsContainer(measures);
-				if (expanded) {
-					return;
-				} else {
-					return this._updateScrollPosition(tabInfo);
+				if (!expanded) {
+					await this._updateScrollPosition(tabInfo);
```

Comment:
This is from some experimentation that I only partially reverted. I can revert this too.